src/components/waste/WasteInputForm.tsx

```diff
@@ -79,6 +79,17 @@ export const WasteInputForm = ({
     ((!isInputAutoFocus && inputValue) || isInputAutoFocus)
   );
 
+  const listContainerHeight = useMemo(
+    () =>
+      (inputValueCitySelected ? filteredStreets : filteredCities)?.length < 6
+        ? device.platform === 'ios'
+          ? 'auto'
+          : (inputValueCitySelected ? filteredStreets : filteredCities)?.length *
+            (normalize(22) + 2 * normalize(16))
```

Comment:
please check this calculation on your emulator and devices

Reply:
but you have not tested the part i mentioned, as you have more than 5 results in the list.